### ISSUE TITLE
[BUG] aria-live should be atomic so content beyond the changing icon gets read out.

### ISSUE THREAD

**Issue by aarongustafson** [aarongustafson/form-validation-list#1]
## Description

Right now only the changing symbol is read (bereft of context) when a validaiton rule changes in status.

## Steps to Reproduce

1. Go to the first demo with VoiceOver on
2. Focus the field — all required validations are read out
3.Type and listen for the changes in the validation status

## Expected Behavior

The symbol _and_ the list item content should be read out

## Actual Behavior

As you type only the symbol changes are read out.

## Additional Context

I think `aria-atomic` is what you want to ensure that the whole list item (including the symbol) is read.
